XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt

```diff
@@ -0,0 +1,19 @@
+<muxc:NavigationView x:Name="nvSample9" 
+                            Grid.Row="1" 
+                            Height="460"
+                            Header="This is Header Text" 
+                            PaneDisplayMode="$(PaneDisplay)" 
+                            IsTabStop="False" 
```

Comment:
This line also feels odd. NavigationView itself isn't a tab stop, is it? So this is redundant?